geode-core/src/main/java/org/apache/geode/pdx/internal/TypeRegistry.java

```diff
@@ -65,6 +65,9 @@
 
   private final InternalCache cache;
 
+  private static final ThreadLocal<Boolean> pdxReadSerializedOverride =
```

Comment:
It's a bit weird to go to all this trouble to use an instance of TypeRegistry with an instance method, only to end up updating a *static* threadlocal. Should this still be static?

Reply:
Thank you.. I overlooked that. Yes, the static is not required anymore.